receiver/rabbitmqreceiver/testdata/integration/expected.3_9.json

```diff
@@ -0,0 +1,89 @@
+{
+   "resourceMetrics": [
+      {
+         "resource": {
+            "attributes": [
+               {
+                  "key": "rabbitmq.queue.name",
+                  "value": {
+                     "stringValue": "some_outgoing_queue"
+                  }
+               },
+               {
+                  "key": "rabbitmq.node.name",
+                  "value": {
+                     "stringValue": "rabbit@localhost"
+                  }
+               },
+               {
+                  "key": "rabbitmq.vhost.name",
+                  "value": {
+                     "stringValue": "/"
+                  }
+               }
+            ]
+         },
+         "scopeMetrics": [
+            {
+               "metrics": [
```

Comment:
Only one metric? Any idea why?

Reply:
@djaglowski I have tried mock some message push and get, I still get one metric, I think the receiver only scrape the current status of the queue, as one of  metric's  description  ——` The number of consumers currently reading from the queue. `   

as for the metric `rabbitmq.message.published` will get right value. the same as `rabbitmq.message.curren`